samples/EndpointRoutingApp/EndpointRoutingApp/Program.fs

```diff
@@ -0,0 +1,63 @@
+﻿open Microsoft.AspNetCore
```

Comment:
I'd like to see some unit tests covering the new endpoint routing code, I guess starting with the examples here. Happy to pick that up after this PR 🙂